meson_options.txt

```diff
@@ -183,6 +183,8 @@ option('fast-bufio', type: 'boolean', value: false,
        description: 'enable some faster buffered i/o operations')
 option('io-wchar', type: 'boolean', value: false,
        description: 'enable wide character support in printf/scanf even when newlib-mb is false')
+option('flockfile', type: 'boolean', value: false,
```

Comment:
This comment is slightly misleading; what the change does is make the I/O operations atomic (as required by the spec). The code is already thread-safe, it just doesn't ensure atomicity and allows I/O from separate threads to interleave.

Reply:
fixed